src/API/Trace/SpanContextInterface.php

```diff
@@ -12,14 +12,11 @@ interface SpanContextInterface
     public const TRACE_FLAG_SAMPLED = 0x01;
     public const TRACE_FLAG_DEFAULT = 0x00;
 
-    /** @todo Implement this in the API layer */
-    public static function createFromRemoteParent(string $traceId, string $spanId, int $traceFlags = self::TRACE_FLAG_DEFAULT, ?TraceStateInterface $traceState = null): SpanContextInterface;
-
     /** @todo Implement this in the API layer */
```

Comment:
Do you think these todo's can be removed now?

Reply:
Yes Since they are implemented in API layer 😳

Reply:
Could you remove them please?